src/mainwindow.cpp

```diff
@@ -189,6 +189,8 @@ MainWindow::MainWindow(QWidget *parent) : QMainWindow(parent), ui(new Ui::MainWi
 	setupUndoRedo();
 
 	restoreGeometry(settings->getValue("geometry", "").toByteArray());
+	if (geometry().y() <= 35)	setGeometry(geometry().x(), 35, geometry().width(), geometry().height());
```

Comment:
Can't me a magic number, find someway to calculate the height of the titlebar.

Reply:
Also, the proper fix wouldn't be to just push it down, it should be to restore the position it was saved. Maybe decode the saved geometry and compare with what it's trying to load back and see what's going awry?